src/ipyradiant/visualization/vis_tool.py

```diff
@@ -0,0 +1,114 @@
+import os
+import textwrap
+from pathlib import Path
+from rdflib import URIRef, Literal, Graph, BNode
+import traitlets as T
+from rdflib.namespace import RDF
+from pandas import DataFrame
+import ipywidgets as W
+from ipycytoscape import MutableDict, CytoscapeWidget
+
+default_edge = {"selector": "edge", "css": {"line-color": "blue"}}
```

Comment:
really gotta watch yourself on making hard style choices for the user. Full-bore blue is very hard to see on black.

You can safely use these [variables](https://github.com/jupyter-widgets/ipywidgets/blob/master/packages/controls/css/labvariables.css). So for example:

```py
{"selector": "edge", "css": {"line-color": "var(--jp-brand-color0)"}}
```

to _usually_ get a blue that works with the default background.

Reply:
sounds good, will do

Reply:
this is cytoscape canvas instructions, not CSS< so not applicable. would be interesting to explore if we end up hacking it harder.